assets/src/edit-story/app/story/storyTriggers/storyEvents/onDirtyRegister.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { useState, useEffect } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { registerPropTypes } from './propTypes';
+import { STORY_EVENTS } from './types';
+
+function isClean(story) {
```

Comment:
nit: not to get stuck on words, but I've found myself being confused by the `clean` / `dirty` lexicon - since here clean would mean blank. could `isNewStory` perhaps be more straight forward?

Reply:
yea I'm not in love with any of the terminology I've been using on here. `isNewStory` or `isBlankStory` both work for me here. 

Any thoughts on the event name? happy to make the evts snake case too if that's the preference

Reply:
maybe `isNewStory` so that it implies the unidirectional nature better?